pkg/minikube/cluster/commands.go

```diff
@@ -16,10 +16,27 @@ limitations under the License.
 
 package cluster
 
-var startCommand = `
-# Run with nohup so it stays up. Redirect logs to useful places.
-PATH=/usr/local/sbin:$PATH nohup sudo /usr/local/bin/localkube start --generate-certs=false > /var/log/localkube.out 2> /var/log/localkube.err < /dev/null &
-`
+import (
+	"fmt"
+	"sort"
+)
 
 // Kill any running instances.
 var stopCommand = "killall localkube | true"
+
+func GetStartCommandWithFlags(flagMap map[string]string) string {
+	startCommandFormattable := `
+# Run with nohup so it stays up. Redirect logs to useful places.
+PATH=/usr/local/sbin:$PATH nohup sudo /usr/local/bin/localkube start %s--generate-certs=false > /var/log/localkube.out 2> /var/log/localkube.err < /dev/null &
```

Comment:
Replace `/var/log/localkube.{out,err}` with arguments too, and replace them with the constants in your other PR after you rebase